Changes

```diff
@@ -1,15 +1,25 @@
-0.03    21st September, 2011
-        Generate a new tarball on linux.  Doing the C<make dist> on OSX 
-        generated a tarball that upset dumb tars
+Revision history for Perl module Number::Compare
+
+0.04    2021-07-?? RCLAMP
```

Comment:
Can we make this HEAD not 0.04, as part of releasing I'll choose the appropriate version to assign.

Reply:
Will do